src/docs/integrations/slack/index.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: "Slack Integration"
+---
+
+Based off of this [post](https://forum.sentry.io/t/how-to-configure-slack-in-your-on-prem-sentry/3463)
+
+## Create a Slack App
+
+To use Sentry’s new Slack integration you’ll need to create an app in Slack bot app. Navigate to __Your Apps__ and click __Create new App__.
+
+![Create](./create.png)
+
+After naming your app and connecting your workspace, navigate to __Basic Information__. Here you’ll find your Client ID, Client Secret, and Verification token that lets your app access the Slack API.
+
+Here’s where you’ll connect your on-premise Sentry instance to your newly created Slack app.
+<dl>
+<dt><code>system.admin-email</code></dt>
+Declared in <code>config.yml</code>.
+
+<dd>
+<markdown>
+Copy your Client ID, Client Secret, and Verification Token.
+
+Set slack.legacy-app to False unless you are supporting an old bot integration (Pre June 2020)
+
+```yaml
+slack.client-id: <'client id'>
```

Comment:
Why do we have quotes around this and not others? 😀

Reply:
i copied verbatim, but i remember this actually helped me initially as this comes in the format 'asd3h3.534b453hg4' so without the quotes it would violate the config (i think?..)